app.js

```diff
@@ -1,4 +1,4 @@
-'user strict';
+'use strict';
```

Comment:
This is super important! Javascript won't throw errors over this, but it also won't do anything unless it's spelled right. Don't learn the hard way like me!